telethon/extensions/markdown.py

```diff
@@ -9,8 +9,8 @@
 from ..tl import TLObject
 from ..tl.types import (
     MessageEntityBold, MessageEntityItalic, MessageEntityCode,
-    MessageEntityPre, MessageEntityTextUrl
-)
+    MessageEntityPre, MessageEntityTextUrl,
+    MessageEntityMentionName)
```

Comment:
Same as my comment above, move to the above line, keep the parenthesis on its own.